docs/configuration.v2alpha1.md

```diff
@@ -123,6 +126,7 @@ spec:
 | global.endpoint.credentials.appKey | AppKey configures your Datadog application key. If you are using clusterAgent.metricsProvider.enabled = true, you must set a Datadog application key for read access to your metrics. |
 | global.endpoint.credentials.appSecret.keyName | KeyName is the key of the secret to use. |
 | global.endpoint.credentials.appSecret.secretName | SecretName is the name of the secret. |
+| global.endpoint.credentials.token | Token is the token for communication between the NodeAgent and ClusterAgent |
```

Comment:
Not sure we need the token field is the common struct as it pushes it down everywhere.

Reply:
do you think it'd be better in ComponentOverride? i'm a bit concerned about confusion around which component one would need to set it in

Reply:
I don't think it should be in `ComponentOverride`, it should be in global but outside of `credentials`

Reply:
i see, ok will do that